src/haxelib/api/GlobalScope.hx

```diff
@@ -21,6 +23,9 @@ using StringTools;
 class GlobalScope extends Scope {
 	function new(repository:Repository) {
 		super(false, repository);
+		
+		// While this class is "GlobalScope", there's currently not a "LocalScope" equivalent class. For now we can add this here. 
+		isLocal = RepoManager.getPath() != RepoManager.getGlobalPath();
```

Comment:
Wouldn't `repository.path` work there?

Reply:
haha yes, I'm still learning my way around the haxelib codebase, will update it to that